app/models/miq_alert_status.rb

```diff
@@ -1,4 +1,19 @@
+require 'ancestry'
+
 class MiqAlertStatus < ApplicationRecord
+  SEVERITY_LEVELS = %w(error warning info).freeze
+
+  has_ancestry
   belongs_to :miq_alert
   belongs_to :resource, :polymorphic => true
+  has_many :miq_alert_status_actions, -> { order "created_at" }, :dependent => :destroy
+  virtual_column :assignee, :type => :string
+
+  def assignee
+    miq_alert_status_actions.where(:action_type => %w(assign unassign)).last.try(:assignee)
```

Comment:
- The order comes from the [association](https://github.com/ManageIQ/manageiq/pull/13233/files/621d518a20fe23020efc2b6b343659b77b1d0712#diff-a826b843e49fc7a2252e620839dd5112R9) there is a test for this [here](https://github.com/ManageIQ/manageiq/pull/13233/files/621d518a20fe23020efc2b6b343659b77b1d0712#diff-207194300e373771bff29999d1406356R73)

- I chose to use unassign since I must have an unacknowledge, for consistency